lms/envs/common.py

```diff
@@ -5455,3 +5455,25 @@ def _should_send_certificate_events(settings):
 derived_collection_entry('EVENT_BUS_PRODUCER_CONFIG', 'org.openedx.learning.certificate.revoked.v1',
                          'learning-certificate-lifecycle', 'enabled')
 BEAMER_PRODUCT_ID = ""
+
+#### Survey Report ####
+# .. setting_name: ENABLE_SURVEY_REPORT
+# .. setting_default: True
+# .. setting_description: Set to True to enable the feature to generate and send survey reports.
+ENABLE_SURVEY_REPORT = True
```

Comment:
How would people feel about renaming this to `SURVEY_REPORT_ENABLE`? I prefer having a common prefix for related settings so that they're easy to find in alphabetically sorted settings files and documentation.

(Non-blocking suggestion.)

Reply:
I also prefer the common prefix and this would apply for `SURVEY_REPORT_ENABLE` and `ANONYMOUS_SURVEY_REPORT`. 
`SURVEY_REPORT_ENABLE` is being introduced by this PR first so it would be easy.
`ANONYMOUS_SURVEY_REPORT` is being set by the [tutor plugin](https://github.com/eduNEXT/tutor-contrib-surveyreport/blob/main/surveyreport/patches/openedx-lms-production-settings)

Reply:
I've renamed the setting to SURVEY_REPORT_ENABLE @timmc-edx @felipemontoya , the setting ANONYMOUS_SURVEY_REPORT, we can do it in other PR because we need to create a PR for tutor plugin also, are you okay with that?

Reply:
Renaming the var should be an easy fix in tutor, but I agree that we should not block this PR longer for this.